### PR TITLE
MMB: Add multicity boarding passes to ticket screen

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/scenes/tickets/boardingPasses/BoardingPassMultiCity.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// @flow strict
+
+import * as React from 'react';
+import { Translation } from '@kiwicom/mobile-localization';
+import { createFragmentContainer, graphql } from '@kiwicom/mobile-relay';
+import { TextIcon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+
+import type { BoardingPassMultiCity as BookingType } from './__generated__/BoardingPassOneWay.graphql';
```
Comment (RobinCsl) [inline app/MMB/src/scenes/tickets/boardingPasses/BoardingPassMultiCity.js]: Is it the right file?

Comment (tbergq) [reply app/MMB/src/scenes/tickets/boardingPasses/BoardingPassMultiCity.js]: Ohh, the dangers of copy/paste I should disable it on my machine 😖

Comment (RobinCsl) [reply app/MMB/src/scenes/tickets/boardingPasses/BoardingPassMultiCity.js]: That's what code reviews are for :) two sets of eyes is better than one ;-)

**Comment by tbergq** [comment]
@RobinCsl updated

**Pull request merged**